### PR TITLE
Rebrand master and K3 from Project Athena to Vircadia.

### PR DESCRIPTION
**Pull request by MarcusLlewellyn**
This rebrands many areas of the interface for Vircadia. Changes include:
- Renaming `athena.sln` to `vircadia.sln`.
- The about dialog has been updated.
- Interface window titles updates.
- Default URL for web entities updated.
- A JSDoc update.
- Updated login screens.
- Updated help dialogs.

### CODE REVIEW

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR356/ProjectAthena-Alpha-PR356-8761f38_e6505baf.dmg

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/kasenvr_project-athena/PR356/ProjectAthena-Alpha-PR356-8761f38_e6505baf.exe

**Comment by vegaslon** [comment]
Could you change BUILD_WIN.md to say vircadia.sln instead of Athena.sln? Save a few questions later on.

**Comment by kasenvr** [comment]
> 
> 
> Could you change BUILD_WIN.md to say vircadia.sln instead of Athena.sln? Save a few questions later on.

I'll make that change in my PR.

**Comment by kasenvr** [comment]
Seems alright to me, though the about dialog doesn't have a centered logo for some reason.

**Comment by kasenvr** [comment]
image
The Vircadia logo on the login screen seems slightly bigger than it should be. ;)

File: interface/resources/html/tabletHelp.html [hunk 1]
```diff
@@ -77,9 +77,9 @@
             var handControllerImageURL = null;
             var index = 0;
             var count = 3;
-            var handControllerRefURL = "https://docs.projectathena.dev/explore/get-started/vr-controls.html#vr-controls";
-            var keyboardRefURL = "https://docs.projectathena.dev/explore/get-started/desktop.html#movement-controls";
-			var gamepadRefURL = "https://docs.projectathena.dev/explore/get-started/vr-controls.html#gamepad";
+            var handControllerRefURL = "https://docs.vircadia.com/explore/get-started/vr-controls.html#vr-controls";
+            var keyboardRefURL = "https://docs.vircadia.com/explore/get-started/desktop.html#movement-controls";
+			var gamepadRefURL = "https://docs.vircadia.com/explore/get-started/vr-controls.html#gamepad";
```
Comment (thoys) [inline interface/resources/html/tabletHelp.html:82]: while touching this line you might as well fix the ident :)

File: interface/src/Application.cpp [hunk 2]
```diff
@@ -3789,8 +3789,8 @@ void Application::setPreferredCursor(const QString& cursorName) {
     if (_displayPlugin && _displayPlugin->isHmd()) {
         _preferredCursor.set(cursorName.isEmpty() ? DEFAULT_CURSOR_NAME : cursorName);
     }
-    else { 
-        _preferredCursor.set(cursorName.isEmpty() ? Cursor::Manager::getIconName(Cursor::Icon::SYSTEM) : cursorName); 
+    else {
```
Comment (thoys) [inline interface/src/Application.cpp:3792]: While touching this line you might also want to fix the structure of this if `else`  
```suggestion
    } else {
```

File: interface/resources/html/tabletHelp.html (hunk 1 above)
Comment (ctrlaltdavid) [inline interface/resources/html/tabletHelp.html:82]:
```suggestion
            var handControllerRefURL = "https://docs.vircadia.dev/explore/get-started/vr-controls.html#vr-controls";
            var keyboardRefURL = "https://docs.vircadia.dev/explore/get-started/desktop.html#movement-controls";
            var gamepadRefURL = "https://docs.vircadia.dev/explore/get-started/vr-controls.html#gamepad";
```

File: interface/src/Application.cpp [hunk 3]
```diff
@@ -1139,7 +1139,7 @@ Application::Application(int& argc, char** argv, QElapsedTimer& startupTimer, bo
     QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Graphik-SemiBold.ttf");
     QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Graphik-Regular.ttf");
     QFontDatabase::addApplicationFont(PathUtils::resourcesPath() + "fonts/Graphik-Medium.ttf");
-    _window->setWindowTitle("Project Athena");
+    _window->setWindowTitle("Vircadia Interface");
```
Comment (ctrlaltdavid) [inline interface/src/Application.cpp:1142]: "Vircadia" or "Vircadia Interface"?
Below, also.

File: interface/src/Menu.cpp [hunk 4]
```diff
@@ -782,21 +782,21 @@ Menu::Menu() {
     // Help/Application menu ----------------------------------
     MenuWrapper * helpMenu = addMenu("Help");
 
-    // Help > About Project Athena
-    action = addActionToQMenuAndActionHash(helpMenu, "About Project Athena");
+    // Help > About Vircadiaa
```
Comment (ctrlaltdavid) [inline interface/src/Menu.cpp:785]:
```suggestion
    // Help > About Vircadia
```

File: interface/src/Menu.cpp [hunk 5]
```diff
@@ -782,21 +782,21 @@ Menu::Menu() {
...
             QString("hifi/dialogs/TabletAboutDialog.qml"), "AboutDialog");
     });
     helpMenu->addSeparator();
 
-    // Help > Athena Docs
+    // Help > Vircadia Docs
     action = addActionToQMenuAndActionHash(helpMenu, "Online Documentation");
     connect(action, &QAction::triggered, qApp, [] {
-        QDesktopServices::openUrl(QUrl("https://docs.projectathena.dev/"));
+        QDesktopServices::openUrl(QUrl("https://apidocs.vircadia.dev/"));
```
Comment (ctrlaltdavid) [inline interface/src/Menu.cpp:796]:
```suggestion
        QDesktopServices::openUrl(QUrl("https://docs.vircadia.dev/"));
```

File: interface/src/Application.cpp (hunk 3 above)
Comment (kasenvr) [reply interface/src/Application.cpp:1142]: Just Vircadia for now.

**Pull request merged**